File: Discussion1/src/car.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    struct {
        string make ="Honda";
        string model ="Civic";

    } car;
    cout << car.make  + " " + car.model + "\n";
    car.model = "Accord";
    cout << car.make + " " + car.model + "\n";
    return 0;
}

File: Discussion1/src/Car.java
public class Car{
    public String make;
    public String model;

    public Car(String make, String model){
        this.make = make;
        this.model = model;
    }

    public static void main(String[] args){
        Car car = new Car("Honda", "Civic");
        System.out.println(car.make + " " + car.model);
        car.model = "Accord";
        System.out.println(car.make + " " + car.model);
    }
}
